pilot/pkg/networking/plugin/mixer/mixer.go

```diff
@@ -223,10 +212,7 @@ func buildOutboundHTTPFilter(mesh *meshconfig.MeshConfig, attrs attributes, node
 					},
 				},
 				MixerAttributes: &mpb.Attributes{Attributes: attrs},
-				ForwardAttributes: &mpb.Attributes{Attributes: attributes{
```

Comment:
I think we still want to forward source UID to the destination.

Reply:
I have guarded the whole thing with a feature flag toggle. See updated code.